test/functional/kaui/admin_allowed_users_controller_test.rb

```diff
@@ -8,9 +8,33 @@ class AdminAllowedUsersControllerTest < Kaui::FunctionalTestHelper
       assert_response :success
     end
 
+    test 'should get create local only' do
+      parameters = {
+          :allowed_user => {:kb_username => 'Albator', :description => 'My french super hero'},
+          :password => 'Albator',
+          :roles => nil,
+          :external => '1'
+      }
+      post :create, parameters
+      assert_equal 'User was successfully configured', flash[:notice]
+      assert_response 302
+    end
+
     test 'should get create' do
-      post :create, :allowed_user => {:kb_username => 'Albator', :description => 'My french super hero'}
+      parameters = {
+          :allowed_user => {:kb_username => 'Albator', :description => 'My french super hero'},
+          :password => 'Albator',
+          :roles => nil
+      }
+      post :create, parameters
+      assert_equal 'User was successfully configured', flash[:notice]
       assert_response 302
+      allowed_user = assigns(:allowed_user)
```

Comment:
`assigns` has been deprecated in Rails 5 (see discussion https://github.com/rails/rails/issues/18950).

While we still have legacy tests relying on it, we probably shouldn't use it for new code. Not quite sure what a good alternative is to verify the view is correctly rendered though 😕